Helper/ProductDescription.php

```diff
@@ -7,6 +7,7 @@ class ProductDescription
     const DISALLOWED_CONTENT = [
         '/<style\b[^>]*>([\s\S]*?)<\/style>/i',
         '/<figure\s+data-content-type="image"[^>]*>[\s\S]*?<\/figure>/i',
+        '/<img\s+src="\{\{media\s+url=[^"]+\}\}"[^>]*\/?>/',
```

Comment:
can you add to https://github.com/storekeeper-company/magento2-plugin/blob/master/Test/Integration/ProductDescriptionTest.php#L8 as well?